modules/coreos-layering-configuring-on-extensions.adoc

```diff
@@ -0,0 +1,81 @@
+// Module included in the following assemblies:
+//
+// * machine_configuration/coreos-layering.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="coreos-layering-configuring-on-extensions_{context}"]
+= Installing extensions into an on-cluster custom layered image
+
+You can install :op-system-first: extensions into your on-cluster custom layered image by creating a machine config that lists the extensions that you want to install. The Machine Config Operator (MCO) installs the extensions into the nodes associated with a specific machine config pool (MCP).
```

Comment:
question: Is `:op-system-first:` supposed to stay like that when rendered? I wasn't seeing anything different when viewing the rich diff. Just wanted to double check whether this is expected

Reply:
Good catch. Thank You!